src/main/java/org/miracum/etl/fhirtoomop/mapper/ConditionMapper.java

```diff
@@ -664,7 +664,7 @@ private List<Pair<String, List<IcdSnomedDomainLookup>>> getValidIcdCodes(
             "ICD Code [{}] in [Condition] {} is not valid in OMOP.",
             uncheckedCode.getCode(),
             conditionId);
-        icdCodeInvalidCounter.increment();
+//        icdCodeInvalidCounter.increment();
```

Comment:
one main comment 
can we uncomment all commented increments ? let the count is taken and then can we handle which reason to be showed and which reasons to be avoided while printing the table